tests/post_training/test_quantize_conformance.py

```diff
@@ -261,142 +429,25 @@ def transform_fn(data_item):
-
-        # quantize OpenVINO model using POT implementation
-        try:
-            def ov_transform_fn(data_item):
-                images, _ = data_item
-                return images.numpy()
-
-            ov_calibration_dataset = nncf.Dataset(batch_one_dataloader, ov_transform_fn)
-
-            ov_model_path = output_folder / (model_name + '.xml')
-            core = ov.Core()
-            ov_model = core.read_model(ov_model_path)
-            ov_quantized_model = nncf.quantize(
-                ov_model, ov_calibration_dataset, **model_quantization_params
-            )
-
-            ov_output_path = output_folder / 'openvino'
-            ov_output_path.mkdir(parents=True, exist_ok=True)
-            q_ov_model_name = model_name + '_ov_int8'
-            q_ov_perf, q_ov_acc = benchmark_ov_model(
-                ov_quantized_model,
-                batch_one_dataloader,
-                q_ov_model_name,
-                ov_output_path,
-            )
-        except Exception as error:
-            traceback_path = Path.joinpath(output_folder, 'openvino', model_name + '_error_log.txt')
-            create_error_log(traceback_path)
-            q_ov_perf = get_error_msg(traceback_path, model_name)
-            q_ov_acc = '-'
-
-        process_connection.send([model_name,
-                                 orig_acc,
-                                 q_torch_acc,
-                                 q_onnx_acc,
-                                 q_ov_native_acc,
-                                 q_ov_acc,
-                                 orig_perf,
-                                 q_torch_perf,
-                                 q_onnx_perf,
-                                 q_ov_native_perf,
-                                 q_ov_perf])
+        for backend in backends:
+            runner = RUNNERS[backend]
+            try:
+                runinfo = runner(model, calibration_dataset, model_quantization_params,
+                                 output_folder, model_name, batch_one_dataloader)
+            except Exception as error:
+                traceback_path = Path.joinpath(output_folder, backend.value, model_name + '_error_log.txt')
+                create_error_log(traceback_path)
+                status = get_error_msg(traceback_path, model_name)
+                runinfo = RunInfo(-1, -1, status)
+            runinfos[backend] = runinfo
+
+        process_connection.send(runinfos)
     except Exception as error:
         traceback_path = Path.joinpath(output_folder, model_name + '_error_log.txt')
         create_error_log(traceback_path)
-        error_message = f'An error occurred while running {model_name}. Traceback: {traceback_path}'
-        result = [model_name, error_message, '-', '-', '-', '-', '-', '-', '-', '-', '-']
-        process_connection.send(result)
+        status = f'An error occurred while running {model_name}. Traceback: {traceback_path}'
+        runinfos[QuantizationBackend.FP32] = RunInfo(-1, -1, status)
```

Comment:
This `except` section was responsible for the whole run of the concrete model, not the concrete backend. So, the message about the error should be widespread to the whole list of backends, isn't it?

Reply:
Yes, and it actually will be widespread, I move this logic over here: https://github.com/openvinotoolkit/nncf/pull/1591/files#diff-a6044884bf87069635cb96de0928ebd1c2d0a1a849693480684139737e35420fR64-R73

So, if `runinfos` has no info about  backends, error message only for FP32 model appears in this row and `-` appears in backend columns